Dockerfile-drupal

```diff
@@ -42,7 +41,6 @@ RUN apt-get update \
   software-properties-common \
   libxml2-dev \
   libcurl4-openssl-dev \
-  imagemagick \
   libmagickwand-dev \
```

Comment:
This can go too, as it's a python binding for Imagemagick (from what I read).

Reply:
Done